corehq/apps/cloudcare/views.py

```diff
@@ -421,6 +423,23 @@ def get(self, request, *args, **kwargs):
         })
 
 
+def get_apps_by_user(domain, user_ids):
+    groups_by_user = get_groups_by_user(domain, user_ids)
+
+    app_access = ApplicationAccess.get_by_domain(domain)
```

Comment:
what if we just sent this down to the browser and did the app filtering in JS using this?